src/reportSources/legacyFflogs/__tests__/__snapshots__/eventAdapter.ts.snap

```diff
@@ -83,19 +83,25 @@ exports[`Event adapter individual events adapts calculateddamage 1`] = `
 Array [
   Object {
     "action": 3554,
-    "sequence": 16647,
```

Comment:
whoops

Reply:
This was an intended change as a result of my updating how PrePullAction synthed... before, the synth was spreading and was pulling forward fields that weren't in the typedef for an action.  Once I had to change how I got the action ID, I went ahead and just explicitly defined only the "right" fields and ditched the spread, thus effecting that test output.

Reply:
haha, yeah, i mean that as in "whoops, that shouldn't've been there in the first place"